users-service/service.yml

```diff
@@ -26,4 +26,6 @@ messages:
 
 
 dependencies:
-  - 'mongo'
+  mongo:
+    prod: 'ds143588.mlab.com:43588'
+    test: 'localhost:27017'
```

Comment:
Actually, we don't even need this to be defined in `service.yml`. `exo-test` is responsible for starting a mongo container that the users service test will run on. This is not something configurable by the user. I will remove this line from `service.yml`.